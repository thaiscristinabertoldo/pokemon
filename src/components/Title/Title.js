import React, { memo } from 'react'

import TitleStyled from './title.css'

export const Title = memo(({ title }) => {
  return <TitleStyled>{title}</TitleStyled>
})
