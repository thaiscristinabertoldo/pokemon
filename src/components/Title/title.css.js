import styled from 'styled-components'

import { COLORS } from '../../constants'

const TitleStyled = styled.h1`
  color: ${COLORS.mediumGrey};
  text-align: center;
  width: 100%;
`

export default TitleStyled
