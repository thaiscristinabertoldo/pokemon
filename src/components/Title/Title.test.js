import React from 'react'

import { render, cleanup } from '@testing-library/react'

import { Title } from './Title'

afterEach(cleanup)

describe('Title', () => {
  it('should take a snapshot', () => {
    const { asFragment } = render(<Title />)

    expect(asFragment(<Title />)).toMatchSnapshot()
  })

  it('should take a Title text', () => {
    const { getByText } = render(<Title title="Title text" />)

    expect(getByText('Title text')).toBeTruthy()
  })
})
