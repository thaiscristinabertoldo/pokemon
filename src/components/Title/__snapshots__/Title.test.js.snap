// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should take a snapshot 1`] = `
<DocumentFragment>
  <h1
    class="sc-AxjAm byhsde"
  />
</DocumentFragment>
`;
